doc-content/jbpm-docs/src/main/asciidoc/Kafka/message-receive-event-proc.adoc

```diff
@@ -3,26 +3,31 @@
 
 When designing your business process in {CENTRAL}, you can create an event that receives Kafka messages.
 
-This event is triggered each time a message arrives in the configured topic. The message is expected to contain data that matches a predefined data object. The {PROCESS_ENGINE} parses the message and provides it as an output of the event.
+This event is triggered each time a message arrives in the configured topic. The message must contain data that matches a predefined data object. The {PROCESS_ENGINE} parses the message and provides it as an output of the event.
+
+.Prerequisites
+
+* You are logged in to {CENTRAL} and have the permission to edit business processes.
+* You created a business process.
 
 .Procedure
 
-. Open the project that contains your business process in {CENTRAL}.
-. Create a data object defining the data that the message will contain. For instructions about creating data objects, see
+. In {CENTRAL}, open the project that contains your business process.
+. Create a data object defining the data that the message must contain. For instructions about creating data objects, see
 ifdef::PAM,DM[]
 {URL_DEVELOPING_PROCESS_SERVICES}#assembly-designing-business-processes[_{DESIGNING_BUSINESS_PROCESSES}_].
 endif::PAM,DM[]
 ifdef::JBPM,DROOLS,OP[]
 xref:jBPMBPMN2[].
 endif::JBPM,DROOLS,OP[]
 . Select the business process and open the business process designer.
-. Add a start event, an intermediate catch event, or a boundary event (attached to a human task) of the type `message`.
+. Add a start event, an intermediate catch event, or a boundary event that is attached to a human task. The event must have the `message` type.
 . Open the properties of the event.
-. In the *Message* field, select *New* and then enter the name of the message. This name must be the same as the name of the topic from which the event is to receive Kafka messages, or else must be defined in an `org.kie.server.jbpm-kafka.ext.topics._broker-topic-name_` {KIE_SERVER} system property.
+. In the *Message* field, select *New* and then enter the name of the message. This name must be the same as the name of the topic from which the event is to receive Kafka messages, or else must be defined as `_topic-name_` in an `org.kie.server.jbpm-kafka.ext.topics._topic-name_` {KIE_SERVER} system property.
 +
 For instructions about using `org.kie.server.jbpm-kafka.ext.topics.*` system properties to define topic names, see xref:kieserver-kafka-proc_{context}[].
 +
-. Add an output data item. Select the data object that you created as its type.
+. Add an output data item and set the type of this data item to the data object that you created.
```

Comment:
[Style suggestion] - Rephrase to remove "of this" and "that you created":

s/Add an output data item and set the type of this data item to the data object that you created./Add an output data item and set the data type to match the data object.

Reply:
I don't think this change works. This is not just any data object, but the data object that the user created at the very start if the process, several steps earlier. We can not link the previous step, so at least we have to say "the data object that you created" explicitly.